test/test-pkg-config.rb

```diff
@@ -233,9 +233,29 @@ def parse_requires(requires)
       @cairo.__send__(:parse_requires, requires)
     end
 
-    def test_broken_version
+    def test_greater_than_or_equals_to_version
       assert_equal(["fribidi"],
                    parse_requires("fribidi >= fribidi_required_dep"))
     end
+
+    def test_greater_than_version
+      assert_equal(["fribidi"],
+                   parse_requires("fribidi > fribidi_required_dep"))
```

Comment:
Please use valid version number instead of `fribidi_required_dep` to focusing on the operator.